freq/lizard_connector.py

```diff
@@ -771,6 +768,24 @@ def from_timeseries_uuid(self, uuid):
         return {}
 
 
+class Users(Base):
```

Comment:
Een class die zelf al een meervoud is qua naam: dat ruikt alsof de class "raar" gebruikt wordt.

Maar.... ik weet niet meer hoe het zat met dat `data_type` gebeuren, dus misschien klopt het wel. Kijk wel even nog goed naar de naamgeving of je het zelf vind kloppen.